SECURITY.md

```diff
@@ -285,6 +285,74 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<RespT> c
 
 [Mutual authentication]: http://en.wikipedia.org/wiki/Transport_Layer_Security#Client-authenticated_TLS_handshake
 
+## Troubleshooting
+
+If you received an error message "ALPN is not configured properly" or "Jetty ALPN/NPN has not been properly configured", it most likely means that:
+ - ALPN related dependencies are either not present in the classpath
+ - or that there is a classpath conflict
+ - or that a wrong version is used due to dependency management.
```

Comment:
For this case, we should tell Maven users to use Maven Enforcer with requireUpperBoundDeps (probably preferred) or dependencyConvergence.

Reply:
Even w/ dependencyConvergence, it may still converge to a wrong version, and/or worse yet, enforcing a newer version of netty, but w/ an incompatible version of tcnative.